core/server/server.js

```diff
@@ -414,14 +432,25 @@ class Server {
     log(`Server is starting up: ${this.baseUrl}`)
 
     const camp = (this.camp = Camp.create({
-      documentRoot: path.resolve(__dirname, '..', '..', 'public'),
+      documentRoot: this.config.public.documentRoot,
       port,
       hostname,
       secure,
       cert,
       key,
     }))
 
+    const index = getHandlerIndex(this.camp, 'staticLayer')
```

Comment:
Could we add an inline comment/link with the relevant scoutcamp info? This feels a bit magical to the non-scoutcamp fluent like me :smile: